pandas/core/frame.py

```diff
@@ -2739,7 +2739,7 @@ def get_value(self, index, col, takeable=False):
 
         Returns
         -------
-        value : scalar value
+        scalar value
```

Comment:
Would be preferable to state the type of scalar being returned here

Reply:
Good question. What you've suggested seems like the most logical thing to me!

cc @datapythonista